apps/server/Dockerfile

```
@@ -1,9 +1,45 @@
-FROM oven/bun:alpine
+FROM oven/bun AS builder
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)